docs/guides/getting-started-orchestrate-apis.md

```diff
@@ -10,6 +10,8 @@ keywords:
 <span class="badge badge--beginner">Beginner</span>
 <span class="badge badge--medium">Time estimate: 15 minutes</span>
 
+import clsx from "clsx";
```

Comment:
@shaarmann because you use clsx in your button configuration, you need an import statement to import the library! I added this in two guides because the HTO guide already had it.

This should fix the build issue once backported! I will take care of that for you now.